variables.tf

```diff
@@ -122,3 +116,20 @@ variable "ignore_changes_on_defaults" {
   type        = bool
   default     = false
 }
+
+variable "context_properties" {
```

Comment:
wdyt about defining one variable with all 'context' related info? with that we can add validation for 'conflicting' variables (`context_template` vs `context_properties`)

Reply:
agreed

Reply:
Decided to continue with separate vars, since it will be easier to pass a map of templates in high level variable (similar to `context.tf` `var.descriptor_formats`)